package.json

```diff
@@ -43,7 +43,9 @@
 		"@nextcloud/dialogs": "^3.0.0",
 		"@nextcloud/event-bus": "^2.0.0",
 		"@nextcloud/l10n": "^1.2.3",
+		"@nextcloud/logger": "^2.0.0",
 		"@nextcloud/router": "^2.0.0",
+		"calendar-js": "git+https://github.com/nextcloud/calendar-js.git",
```

Comment:
this dependency is not ideal but there is only one module that is used in here so I *hope* it won't bloat our bundle sizes